src/AcceptanceTests/TestSuiteConstraints.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace NServiceBus.AcceptanceTests
+{
+    using AcceptanceTesting.Support;
+
+    public partial class TestSuiteConstraints
+    {
+        public bool SupportsDtc { get; } = false;
+        public bool SupportsCrossQueueTransactions { get; } = true; // TODO: verify that this is an AtomicSendWithReceive. If not, set to false
```

Comment:
@SeanFeldman this is the only test that uses it https://github.com/Particular/NServiceBus/blob/f00c76bc26e70d6e5188473f79e38e51d7445927/src/NServiceBus.AcceptanceTests/Tx/When_receiving_with_native_multi_queue_transaction.cs#L11 

It is literally can a transaction spawn multiple queues

https://github.com/Particular/NServiceBus/blob/f00c76bc26e70d6e5188473f79e38e51d7445927/src/NServiceBus.AcceptanceTests/Tx/When_receiving_with_native_multi_queue_transaction.cs#L39